source/core/NumericString.cpp

```diff
@@ -224,11 +224,14 @@ void DefaultFormatCode(Int32 count, StaticTypeAndData arguments[], TempStackCStr
         }
         switch (argType->BitEncoding()) {
 
-        case kEncoding_UInt:
-        case kEncoding_Enum: {
+        case kEncoding_UInt: {
             buffer->AppendCStr("%u");
         }
         break;
+            case kEncoding_Enum: {
```

Comment:
It lines up in the actual code and that line wasn't touched by me, so I didn't notice; it doesn't here because of the + added by the differ.
But since you ask, I'm going to go ahead and remove ALL the extraneous tabs in the code.